client/webserver/site/src/js/markets.ts

```diff
@@ -1972,14 +2023,11 @@ export default class MarketsPage extends BasePage {
       order: wireOrder(order),
       pw: pw
     }
-    if (!this.validateOrder(order)) return
```

Comment:
This validation has no effect since user can't modify any of the values on `verifyForm` page and `orderForm` page already calls this func